ui/src/components/Organization/List.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames/bind';
+import style from 'src/components/Organization/style.module.scss';
+
+const c = classNames.bind(style);
+
+interface Organizations {
+  id: string;
+  name: string;
+  banner?: string;
+  square_logo?: string;
+}
+
+interface FieldProps {
+  list: Organizations[];
```

Comment:
`list` is a very generic name. A variable's name should represent the intent behind its existence; ideally it should reflect what's stored in it, and what it might be used for. A better name here would be `organizations`.